docs/developers/connect-sdk.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: CosmWasm Connect SDK
+description: Querying Connect prices using CosmWasm and Connect SDK
+icon: atom-simple
+---
+
+<Note>
+**Building with Connect? Join our [Discord](https://discord.gg/amAgf9Z39w)**!
+</Note>
+
+The Connect SDK provides bindings for the `x/oracle` and `x/marketmap` modules.
+
+## Adding Connect SDK To Your Contract
+
+The version of the Connect SDK for your contract depends on the Connect protocol version of the chain.
+
+| Connect Protocol Version | Connect SDK Version                   |
+|--------------------------|---------------------------------------|
+| v1.x                     | v0.1.0                                |
+| v2.x                     | v0.2.0                                |
+
+<Tip>
+    How to find the Connect Protocol version
+
+    The protocol version of Connect can be found on the chain by either:
+
+    1. Checking the chain's `go.mod` file.
+    2. Checking the required version in the [quickstart](../validators/quickstart#run-connect-sidecar)
+</Tip>
+
+Add the following line to the `dependencies` section of your `Cargo.toml`:
+
+`connect-sdk = { git = "https://github.com/skip-mev/connect-sdk", tag = "CONNECT SDK VERSION HERE", package = "connect-sdk" }`
+
+
+
+## Safely Accessing Price Data
+
+The following checks should be made before utilizing a price from Connect in a contract:
+
+1. currency-pair exists within the `x/oracle` and `x/marketmap` modules.
+2. currency-pair is `enabled` within the `x/marketmap`.
+3. price `block_height` is not older than a few blocks.
+4. price nonce is not 0.
+
+
+### Code Example
+
+```rust contract.rs
+use connect_sdk::bindings::query::ConnectQuery;
+use connect_sdk::bindings::querier::ConnectQuerier;
+use connect_sdk::bindings::marketmap::types::CurrencyPair;
+use cosmwasm_std::{entry_point, Binary, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdResult, to_json_binary, Int128, StdError};
+
+fn do_something_with_price(
+    deps: Deps<ConnectQuery>,
+    env: Env,
+    currency_pair: CurrencyPair
+) -> StdResult<Int128> {
+    let connect_querier = ConnectQuerier::new(&deps.querier);
+    let base = currency_pair.base.to_uppercase();
+    let quote = currency_pair.quote.to_uppercase();
+
+    // Check if the market exists and is enabled
+    let market = connect_querier.get_marketmap_market(base.clone(), quote.clone())?;
+    if !market.market.ticker.enabled {
+        return Err(StdError::generic_err("market is not enabled"));
+    }
+
+    // Check price validity
+    let price_response = connect_querier.get_oracle_price(base, quote)?;
```

Comment:
error[E0061]: this method takes 1 argument but 2 arguments were supplied
  --> contracts/x-oracle-passthrough/src/contract.rs:22:42
   |
22 |     let price_response = connect_querier.get_oracle_price(base, quote)?;

Reply:
ah, im using v0.1.0 for the example, likely an error from v0.2

Reply:
ill add that im using that version to the doc